.gitmodules

```diff
@@ -1,7 +1,3 @@
 [submodule "vendor/polkavm"]
 	path = vendor/polkavm
 	url = https://github.com/acalanetwork/polkavm
```

Comment:
Should we remove vendored PolkaVM code as well?

Reply:
No, needed for the `pallet-qf-polkavm`. The patch addresses `polkadot-sdk` dependency only.